pom.xml

```diff
@@ -1470,5 +1470,31 @@
       </build>
     </profile>
 
+    <profile>
+      <id>pitest-size</id>
```

Comment:
Because of pitest separation, we will most likely need to write a junit to verify we always have all packages added to pitest. We should rename this to follow "pitest-" plus the package name. I think we should rename this to `pitest-sizes` or `pitest-checks-sizes` depending on if we are going to run this on non-check classes like API, Main, etc.